mlrun/model.py

```diff
@@ -281,6 +378,18 @@ def __init__(
 
 
 class BaseMetadata(ModelObj):
+    _fields_to_strip = [
```

Comment:
I think you are missing this
```suggestion
    _fields_to_strip = ModelObj._fields_to_strip + [
```